### ISSUE TITLE
Wrong decorator mentioned in docs/components/pipelines/user-guides/components/containerized-python-components.md

### ISSUE THREAD

**Issue by ypitrey** [kubeflow/website#3939]
I got to this page from the page on migrating from v1 to v2. There's a link there that reads "Change: Use the [@dsl.container_component](https://kubeflow-pipelines.readthedocs.io/en/stable/source/dsl.html#kfp.dsl.container_component) decorator as described in [Container Components](https://www.kubeflow.org/docs/components/pipelines/user-guides/components/containerized-python-components/)."

But this page does not mention the `@dsl.container_component` decorator, only the `@dsl.component` decorator. This is confusing 😅

**Comment by varodrig** [comment]
@ypitrey would you like to contribute by submitting a PR with this change?

**Comment by siddarth2810** [comment]
is this the one ? the link to the  below text is [here](https://www.kubeflow.org/docs/components/pipelines/user-guides/migration/) but I dont see any link coming to this page

![Image](https://github.com/user-attachments/assets/2dab4201-8203-4878-9b93-ba510adb3d90)

**Comment by varodrig** [comment]
> is this the one ? the link to the below text is [here](https://www.kubeflow.org/docs/components/pipelines/user-guides/migration/) but I dont see any link coming to this page
> 
> ![Image](https://github.com/user-attachments/assets/2dab4201-8203-4878-9b93-ba510adb3d90)

yes, exactly it's mentioned to use the decorator but it's not reflected on the documentation

**Comment by varodrig** [comment]
/transfer pipelines
cc @hbelmiro
